elide-core/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -157,46 +153,35 @@ private Object fetchObjects(Environment context) {
      */
     private Class getRecordType(RequestScope requestScope, GraphQLType outputType) {
         GraphQLObjectType graphQLType = (GraphQLObjectType) ((GraphQLList) outputType).getWrappedType();
+        String entityName = graphQLType.getName();
```

Comment:
This logic (using the name of the graphQL type to get the underlying class name) will only work for GraphQLOutputType.  Change the outputType parameter to match this constraint.